test/extended/apiserver/resiliency.go

```diff
@@ -52,6 +57,14 @@ var _ = ginkgo.Describe("[Conformance][sig-sno][Serial] Cluster", func() {
 		framework.ExpectNoError(err)
 		gomega.Expect(clusterApiServer.Status.NodeStatuses[0].TargetRevision).To(gomega.Equal(int32(0)))
 
+		ginkgo.By("Checking pods restart count")
```

Comment:
this does not fit to the test string "allow a fast rollout". People will assume the API is at fault while the workloads or kubelet are.

Reply:
The test first validates that the disruption caused by kube-apiserver rollout is lower than allowed downtime.
It's true that I'm using this test to validate that the pods (in the openshift-* and kube-* namespaces) don't get restarted during the kube-apiserver rollout.
  
Rephrased this line to say: `ginkgo.By("Initializing pods restart count")` 
Added: `ginkgo.It("with no pods restarts during API disruption", func() {` before validating the pods restarts, so that in case it fails due to the pods restarts it would show up as:
`[Conformance][sig-sno][Serial] Cluster should allow a fast rollout of kube-apiserver with no pods restarts during API disruption`